lib/rihanna_ui_web/controllers/jobs_controller.ex

```diff
@@ -26,4 +26,11 @@ defmodule RihannaUIWeb.JobsController do
     end
     |> redirect(to: "/failed")
   end
+
+  def mutate(conn, %{"action" => "retry_all"}) do
+    RihannaUI.Job.failed()
+    |> Enum.map(&Rihanna.retry(&1.id))
```

Comment:
This is gonna be pretty slow for many hundreds of jobs.

If you feel like it, you could modify Rihanna's `retry` function to take a list of jobs as well and do it with one SQL query.